### ISSUE TITLE
numeric type problem

### ISSUE THREAD

**Issue by paolino** [nikita-volkov/hasql-th#14]
I'm not sure I got the syntax right
Is there anything wrong here ?
```haskell
getSamples' =
        [vectorStatement| 
            select time :: int8
            , b_in :: numeric
            , b_out :: numeric
            , line_id :: int4? 
            , hostname :: text
            , interface :: text
            , description :: text
            , accountable :: bool
                  from samples where
                      time >= $1 :: int8
                      and line_id is not null
                      and (b_in > 0 or b_out > 0)
                      order by time limit $2 :: int8
              |]
```
I get 

```

  |
3 |             , b_in :: numeric
  |                              ^
Reserved keyword "numeric" used as an identifier. If that's what you intend, you have to wrap it in double quotes.
```

**Comment by nikita-volkov** [comment]
Try running with the latest version of "hasql-th", there was a bug in previous releases, which is already fixed

**Comment by paolino** [comment]
It's using hasql-th-0.4.0.6, I will try HEAD

**Comment by paolino** [comment]
Just as confirmation
```haskell
getSamples' =
        [vectorStatement| 
            select time :: int8
            , line_id :: int4? 
            , hostname :: text
            , interface :: text
            , description :: text
            , accountable :: bool
                  from samples where
                      time >= $1 :: int8
                      and line_id is not null
                      and (b_in > 0 or b_out > 0)
                      order by time limit $2 :: int8
              |] 
```
compiles

**Comment by nikita-volkov** [comment]
Thanks. Should be fixed in 0.4.0.7
